modules/auxiliary/gather/ldap_query.rb

```diff
@@ -440,6 +461,7 @@ def run
           unless (@base_dn = discover_base_dn(ldap))
             print_warning("Couldn't discover base DN!")
           end
+          @schema_dn = find_schema_dn(ldap)
```

Comment:
If the user specifies their own `datastore['BASE_DN']` - would that make `@schema_dn` nil still?

Reply:
Woops good spotting, let me fix that up.

Reply:
Fixed in https://github.com/rapid7/metasploit-framework/pull/17532/commits/71aa4bdace154ef9523a78427be61030d7d867f5